packages/config/config/devices/0x014a/tstat-tbz500.json

```diff
@@ -0,0 +1,625 @@
+			"valueSize": 2,
+			"defaultValue": 1
+		},
+		{
+			"#": "23[0x0080]",
+			"label": "Send Status Change Report: Schedule Enable",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"valueSize": 2
+		},
+		{
+			"#": "23[0x0100]",
+			"label": "Send Status Change Report: Setback",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"valueSize": 2
+		},
+		{
+			"#": "23[0x0200]",
+			"label": "Send Status Change Report: Run Hold",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"valueSize": 2
+		},
+		{
+			"#": "23[0x0400]",
+			"label": "Send Status Change Report: Display Lock",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"valueSize": 2
+		},
+		{
+			"#": "23[0x2000]",
+			"label": "Send Status Change Report: Battery",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"valueSize": 2,
+			"defaultValue": 1
+		},
+		{
+			"#": "23[0x4000]",
+			"label": "Send Status Change Report: Mechanical Status",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"valueSize": 2
+		},
+		{
+			"#": "23[0x8000]",
+			"label": "Send Status Change Report: SCP Status",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"valueSize": 2
+		},
+		{
+			"#": "24",
+			"label": "Display Lock",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Unlocked",
+					"value": 0
+				},
+				{
+					"label": "Locked",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "26",
+			"label": "Backlight Timer",
+			"unit": "seconds",
+			"valueSize": 1,
+			"minValue": 10,
+			"maxValue": 30,
+			"defaultValue": 20
+		},
+		{
+			"#": "33",
+			"label": "Max Heat Setpoint",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 30,
+			"maxValue": 109,
+			"defaultValue": 90
+		},
+		{
+			"#": "34",
+			"label": "Min Cool Setpoint",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 33,
+			"maxValue": 112,
+			"defaultValue": 60
+		},
+		{
+			"#": "38",
+			"label": "Schedule Enable",
+			"$import": "~/templates/master_template.json#base_enable_disable"
+		},
+		{
+			"#": "40",
+			"label": "Setback Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "No Setback",
+					"value": 0
+				},
+				{
+					"label": "Unoccupied Mode",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "41",
+			"label": "Unoccupied HSP",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 30,
+			"maxValue": 109,
+			"defaultValue": 62
+		},
+		{
+			"#": "42",
+			"label": "Unoccupied CSP",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 33,
+			"maxValue": 112,
+			"defaultValue": 80
+		},
+		{
+			"#": "43",
+			"label": "Remote Sensor 1 Node Number",
```

Comment:
Not sure what this one does, maybe it needs a short description.

Reply:
Yes, I will add one. The devices allows use of a secondary temperature sensor (this specifies the ZW node) that can be used instead of (or in addition to), the one built into the unit. Haven't tried it yet.